lib/IRGen/GenClangType.cpp

```diff
@@ -32,10 +32,6 @@ clang::CanQualType IRGenModule::getClangType(CanType type) {
 }
 
 clang::CanQualType IRGenModule::getClangType(SILType type) {
-  if (type.isForeignReferenceType())
-    return getClangType(type.getASTType()
-                            ->wrapInPointer(PTK_UnsafePointer)
-                            ->getCanonicalType());
```

Comment:
This is now handled inside of `getClangTypeForIRGen`, so this logic was adding an (incorrect) extra pointer.